tests/python/rest_api/test_tasks.py

```diff
@@ -1518,3 +1521,64 @@ def test_can_report_correct_completed_jobs_count(tasks, jobs, admin_user):
 
         task, _ = api_client.tasks_api.retrieve(task["id"])
         assert task.jobs.completed == 1
+
+
+class TestImportTaskAnnotations:
```

Comment:
Shouldn't there be tests for:
- 2 parallel uploads, if it is supported now
- a successful uploading after an interrupted upload
- the file cleanup after successful and unsuccessful uploads (I don't think we need to store unfinished files longer than, e.g., 24 hours)

Reply:
> the file cleanup after successful and unsuccessful uploads (I don't think we need to store unfinished files longer than, e.g., 24 hours)

What do you mean by `no need to store unfinished files longer than, e.g., 24 hours`? Now all uploaded files will be deleted in any case (it doesn't matter how the job was finished) immediately after completion.

Reply:
After completion of what? The uncompleted uploads (e.g., a half of the file(-s) was uploaded, then the browser was closed) will never trigger the job.